oddNumbers.js

```diff
@@ -1,13 +1,41 @@
-const nums = [];
+let randNums=[];
 
-function oddNumbers() {
+document.getElementById("showing5ran").addEventListener("click",show5ranNums);
+document.getElementById("showing3largest").addEventListener("click", show3largNums);
 
-    for (let i = 1; i < 100; i++) {
-        if (i % 2 !== 0) {
-            nums.push(i);
-        }
+function show5ranNums(){
+    for(let i=0; i<10;i++){
+        let temp=Math.floor(Math.random() * 10+1);
+    if(randNums.includes(temp)==false){
+        randNums.push(temp);
+    }else{
+        i--;
     }
+    }
+    let temp2="";
+    for(let j=0; j<10;j++){
+temp2+=randNums[j]+", ";
+    }
+    let newStr = temp2.slice(0, -2);
+    document.getElementById("5ranNums").innerHTML=newStr;
 }
-oddNumbers();
 
-console.log(nums);
+function show3largNums(){
```

Comment:
Function name need to be showLargest for example, avoid numbers in function names